app/models/study_type.rb

```diff
@@ -0,0 +1,38 @@
+# == Schema Information
+#
+# Table name: study_types
+#
+#  id          :integer          not null, primary key
+#  name        :text             not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  description :text
+#
+# Indexes
+#
+#  index_study_types_on_name  (name) UNIQUE
+#
+
+class StudyType < ActiveRecord::Base
+  # The name we expect the "other" record to have in the db
+  OTHER_STUDY_TYPE_NAME = "Other".freeze
+
+  has_many :studies, inverse_of: :study_type
+
+  validates_presence_of :name
+  validates_uniqueness_of :name
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.